src/invidious/views/channel.ecr

```diff
@@ -21,7 +21,7 @@
         </div>
     </div>
     <div class="pure-u-1-3" style="text-align:right">
-        <h3>
+        <h3 style="text-align:right">
```

Comment:
you'll have to add that change to the other channel pages, too (`community.ecr` and `playlists.ecr`). And also remove the duplicate `style` on the `<div>` right above :)

Reply:
I kept the alignment on div since I thought if there be more elements in it, it should be able to handle it properly.

I checked playlist. it didn't need since it doesn't use `h3`. There is inconsistency!

Cannot find when community view gets used.

Reply:
Though I couldn't find community page, I added the style to relevant element there as well.

Reply:
> I checked playlist. it didn't need since it doesn't use `h3`. There is inconsistency!

It's `playlists.ecr`, with a `s` ! The file `playlist.ecr` shows a single playlist, where the file `playlists.ecr` show all the playlists that a channel has.

> Cannot find when community view gets used.

See here at the bottom? this is the "community" tab. This is also here that you find the channel playlists :)

![image](https://user-images.githubusercontent.com/52980486/123271264-f14cf180-d500-11eb-9ee6-973394dbb055.png)

Reply:
I know, this is not very obvious (both the files naming and the links position) but we're working on it ^^